File: src/apps/Andriod/app/src/main/java/com/dkellycollins/triggerfinger/util/MathUtil.java
package com.dkellycollins.triggerfinger.util;

public class MathUtil {

    public static float distanceSquared(float x1, float x2, float y1, float y2) {
        float x = x1 - x2;
        float y = y1 - y2;

        return x*x + y*y;
    }

}

File: src/apps/Android-Kotlin/app/src/main/kotlin/com/dkellycollins/triggerfinger/util/MathUtil.kt
package com.dkellycollins.triggerfinger.util

object MathUtil {

    fun distanceSquared(x1: Float, x2: Float, y1: Float, y2: Float): Float {
        val x = x1 - x2
        val y = y1 - y2

        return x * x + y * y
    }

}
